File: Linguagem C++/Vetores/SomaVetores.cpp
#include <iostream>

using namespace std;

int main()
{
    int n;

    cout << "Quantos valores vai ter cada vetor? ";
    cin >> n;

    int vetA[n];
    int vetB[n];
    int vetC[n];

    // Lendo o vetor A
    cout << "Digite os valores do vetor A:" << endl;
    for (int i=0;i<=n-1;i++){
        cin >> vetA[i];
    };

    // Lendo o vetor B
    cout << "Digite os valores do vetor B:" << endl;
    for (int i=0;i<=n-1;i++){
        cin >> vetB[i];
    };

    // Somando os vetores A e B
    for (int i=0;i<=n-1;i++){
        vetC[i] = vetA[i] + vetB[i];
    };

    // Imprimindo o vetor C
    cout << "VETOR RESULTANTE:" << endl;
    for (int i=0;i<=n-1;i++){
        cout << vetC[i] << endl;
    };

    return 0;
}

File: Linguagem Java/Exercicios/SomaVetores.java
import java.util.Locale;
import java.util.Scanner;

public class SomaVetores {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);

		int n;

		System.out.print("Quantos valores vai ter cada vetor? ");
		n = sc.nextInt();

		int vetA[] = new int[n];
		int vetB[] = new int[n];
		int vetResultante[] = new int[n];

		// Lendo o vetor A
		System.out.println("Digite os valores do vetor A:");
		for (int i = 0; i < n; i++) {
			vetA[i] = sc.nextInt();
		};

		// Lendo o vetor B
		System.out.println("Digite os valores do vetor B:");
		for (int i = 0; i < n; i++) {
			vetB[i] = sc.nextInt();
		};
		
		// Soma dos vetores		
		for (int i=0;i<n;i++) {
			vetResultante[i] = vetA[i] + vetB[i];			
		};
		
		// Imprimindo vetor resultante
		System.out.println("VETOR RESULTANTE:");
		for (int i=0;i<n;i++) {
			System.out.println(vetResultante[i]);			
		};
		
		sc.close();
		

	}

}
